test/fixtures/output/login/1.golden

```diff
@@ -1,4 +1,4 @@
 - is_current: true
-  name: login-on-prem@example.com-http://127.0.0.1:\d+\?cacertpath=.+test[/\\]fixtures[/\\]input[/\\]login[/\\]test\.crt
+  name: login-on-prem@example.com-http://127.0.0.1:\d+\?cacertpath=.*test[/\\]fixtures[/\\]input[/\\]login[/\\]test\.crt
```

Comment:
I'm curious about this change... why couldn't this be empty before but now it can?